lib/tasks/demo_data_for_development.rake

```diff
@@ -0,0 +1,63 @@
+        create(:event_schedule, event: second_scheduled_event, schedule: demo_schedule, start_time: conference.start_date + conference.start_hour.hours + 30.minutes, room: conference_rooms.third)
+        create(:event_schedule, event: multiple_speaker_event, schedule: demo_schedule, start_time: conference.start_date + conference.start_hour.hours, room: conference_rooms.first)
+
+        create(:registration, user: user1, conference: conference)
+        create(:registration, user: user2, conference: conference)
+    end
+
+    conference = create(:full_conference, title: 'Jangouts Demo Conference', short_title: 'jangouts', start_hour: 15, start_date: 7.days.ago, description: "This is a full conference demo instance happened in the past. It contains open cfp, venue/rooms, submitted talks by multiple speakers, partly confirmed talks and multiple schedules.")
+    generate_program conference
+    conference.program.cfp.update_attributes(start_date: 4.days.ago, end_date: 2.days.ago)
+    conference.update_attributes(end_date: 1.day.ago)
+    conference.registration_period.update_attributes(start_date: 9.days.ago, end_date: 8.days.ago)
+
+    conference = create(:full_conference, title: 'Open Source Event Manager Demo Conference', short_title: 'osemdemo', start_hour: 8, description: "This is a full conference demo instance happening in the future. It contains open cfp, venue/rooms, submitted talks by multiple speakers, partly confirmed talks and multiple schedules.")
+    generate_program conference
+
+    registration_period = create(:registration_period)
+    conference = create(:conference, title: 'Portus Demo Conference', short_title: 'portus', registration_period: registration_period, end_hour: 15, description: "This is a demo conference instance. No information about this conference is set by default.")
+    create(:event, program: conference.program, title: 'Demo Event', abstract: 'This is a demo event instance whose state not defined.')
+    create(:event, program: conference.program, title: 'Demo Rejected Event', state: 'rejected', abstract: 'This is demo event instance in a rejected state.')
+    create(:event, program: conference.program, title: 'Demo Unconfirmed Event', state: 'unconfirmed', abstract: 'This is a demo event instance in unconfirmed state.')
+    create(:event, program: conference.program, title: 'Demo Confirmed Event', state: 'confirmed', abstract: 'This is a demo event instance in a confirmed state.')
+
+    registration_period = create(:registration_period, start_date: 4.days.ago, end_date: 2.days.ago)
+    conference = create(:full_conference, title: 'Yast Demo Conference', short_title: 'yast' , registration_period: registration_period, end_hour: 23, start_date: 1.day.ago, description: "This is a full conference demo instance. Its registration period is closed.")
```

Comment:
Please use single quotes for the description as well, and anywhere else unless we have string interpolation.